engine/glfw/java/com/dynamo/android/DefoldActivity.java

```diff
@@ -126,9 +121,12 @@ public boolean isAppInstalled(String appPackageName) {
      */
     public native void FakeBackspace();
     public native void FakeEnter();
+    public native void glfwInputBackButton();
     public native void glfwInputCharNative(int unicode);
     public native void glfwSetMarkedTextNative(String text);
 
+    private boolean keyboardActive;
```

Comment:
it's better to handle keyboard on java side since we get Back event on java side. Otherwise, the engine will crash because of Java->Cpp->Java call and to avoid it  we will need another synchronization point at Java->Cpp->sync->Java